src/components/PersonElement.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { NavLink, useLocation } from 'react-router-dom';
+import { Person } from '../types/Person';
+
+interface Props {
+  person: Person,
+  handleSelection: (slug: string,) => void,
+  personSlug: string | undefined,
+}
+
+export const PersonElement: React.FC<Props> = ({
+  person,
+  handleSelection,
+  personSlug,
+}) => {
+  const {
+    name,
+    sex,
+    born,
+    died,
+    fatherName,
+    motherName,
+    slug,
+    mother,
+    father,
+  } = person;
+
+  const location = useLocation().pathname.toString().slice(8);
```

Comment:
What about using [useParams](https://reactrouter.com/en/main/hooks/use-params) here and giving more clear name to this const?)